web_src/less/variables.less

```diff
@@ -0,0 +1,21 @@
+// here a standard set of media queries is defined, that is compatible with the
+// responsive grid used in formantic css. As there only is a precompiled build
+// of formantic checked in (since 946bbbe),we can't use their less variables
+// here, but these breakpoints shouldn't change in the foreseable future.
+
+@breakSmMax: ~"(max-width: 767px)";
+@breakMdMin: ~"(min-width: 768px)";
+@breakMdMax: ~"(max-width: 991px)";
+@breakLgMin: ~"(min-width: 992px)";
+@breakLgMax: ~"(max-width: 1200px)";
+@breakXlMin: ~"(min-width: 1201px)";
```

Comment:
Do we really need all those variables? I'd prefer to always use `max-width` and never `min-width` with just the breakpoint in pixels in the variables because I don't want to really look up the variables every time I want to use a media query so simplicity is important imho. I'd do something like this, following the 5 Fomantic breakpoints:

```less
@breakXs: 319px;
@breakSm: 767px;
@breakMd: 991px;
@breakLg: 1199px;
@breakXl: 1919px;
```

Reply:
Well, these min-width and max-width contraints are used across the stylesheet..
If you don't want to refactor all of that, we need at least the two varaibles for each breakpoint as to avoid the issue that prompted this PR in the first place.

I guess we can rename, so it's easy to remember: the combined variables, so you'd write:
```less
@mediaSmAndUp: ~"(min-width: 768px)";
@mediaSmAndDown: ~"(max-width: 991px)";
@mediaLgAndUp: ~"(min-width: 992px)";
@mediaLgAndDown: ~"(max-width: 1200px)";
@mediaLgAndUp: ~"(min-width: 1201px)";

@mediaXs: ~"(max-width: 767px)";
@mediaSm: ~"@mediaSmAndUp and @mediaSmAndDown";
...
```

Reply:
It needs to be refactored, but I guess that can happen later.